lib/buffer.js

```diff
@@ -933,8 +940,12 @@ Buffer.prototype.readUInt32BE = function(offset, noAssert) {
 
 
 Buffer.prototype.readIntLE = function(offset, byteLength, noAssert) {
+  if (!noAssert)
+    checkByteLength(byteLength);
```

Comment:
@seppevs I can think of good arguments for both sides, so I think we can wait for thoughts from other collaborators.